src/api/app/views/webui/patchinfo/show.html.haml

```diff
@@ -41,7 +41,10 @@
       - if @issues.present?
         %ul
           - @issues.each do |issue|
-            = issue_link(issue)
+            - if issue[0] == "CVE"
+              = content_tag(:li, link_to((issue[1]).to_s, issue[2]) + ": #{issue[3]}")
```

Comment:
Why content_tag? and not something like ` %li= link_to((issue[1]).to_s, issue[2]) + ": #{issue[3]}"`